perses/annihilation/relative.py

```diff
@@ -2527,9 +2534,6 @@ def _handle_nonbonded(self):
         hybrid_to_old_map = self._hybrid_to_old_map
         hybrid_to_new_map = self._hybrid_to_new_map
 
-        #create the forces...
-        self._hybrid_system_forces['NonbondedForce'] = openmm.NonbondedForce()
```

Comment:
Why is this not needed anymore? Was it ever needed?

Reply:
If you search for `self._hybrid_system_forces['NonbondedForce']` in the file, its never used anywhere. I don't think it was ever needed.